C++.en.md

```diff
@@ -0,0 +1,146 @@
+Wiren Board C++ Style Guide
+========================
+
+The following sheet is based on Yandex code style for C++. If any guideline is missing from the following document, 
+please follow Yandex official documentation: https://clickhouse.yandex/docs/en/development/style.html . But if you
```

Comment:
looks like this document is made by another team and thus different...
The one our conventions were based at is called Arcadia C++ Style Guide, and I'm afraid it's not public

Reply:
on a second thought, it's very closely related to that Arcadia one, so please keep it